docs/index.asciidoc

```diff
@@ -298,6 +299,22 @@ A value less than zero is a configuration error.
 
 The amount of time to wait before attempting to retry a failed produce request to a given topic partition.
 
+[id="plugins-{type}s-{plugin}-sasl_jaas_config"]
+===== `sasl_jaas_config` 
+
+  * Value type is <<string,string>>
+  * There is no default value for this setting.
+
+The plugin local JAAS configuration setting which is not java process shared as setting in `jaas_path` config file. This allows each plugin instance to has its own configuration. If both `sasl_jaas_config` and `jaas_path` configurations are set, the setting here takes precedence.
+
+Example (setting for Azure Event Hub):
+[source,ruby]
+    output {
+      kafka {
+        sasl_jaas_config => "org.apache.kafka.common.security.plain.PlainLoginModule required username='$ConnectionString' password='Endpoint=sb://EVENTHUBNAME.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=SOMEKEY';"
```

Comment:
Could you change the username and password of the sample `sasl_jaas_config` setting to something like `username='auser' password='asecret'` as opposed to the example for `Azure Event Hub` use? I suspect that that use case is fairly unusual for users of this plugin, and a more general example would be helpful:
```suggestion
        sasl_jaas_config => "org.apache.kafka.common.security.plain.PlainLoginModule required username='auser'  password='apassword';"
```